File: app/src/main/java/com/example/android/popularmovies/rest/ApiInterface.java
package com.example.android.popularmovies.rest;

import com.example.android.popularmovies.model.MoviesResponse;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface ApiInterface {

    @GET("movie/popular")
    Call<MoviesResponse> getPopularMovies(@Query("api_key") String apiKey , @Query("page") int page);
}

File: app/src/main/kotlin/com/example/android/popularmovies/rest/ApiInterface.kt
package com.example.android.popularmovies.rest

import com.example.android.popularmovies.model.MoviesResponse

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface ApiInterface {

    @GET("movie/popular")
    fun getPopularMovies(@Query("api_key") apiKey: String): Call<MoviesResponse>
}
